packages/x-components/src/design-system/components/input-group/default.scss

```
@@ -74,15 +76,22 @@
     flex: 1 1 100%;
     min-width: 0;
     border: none;
+  }
 
-    &::placeholder {
-      // typography
-      color: var(--x-color-text-input-group-placeholder-default);
-      font-family: var(--x-font-family-input-group-placeholder-default);
-      font-size: var(--x-size-font-input-group-placeholder-default);
-      font-weight: var(--x-number-font-weight-input-group-placeholder-default);
-      line-height: var(--x-size-line-height-input-group-placeholder-default);
-    }
+  > .x-input-placeholder {
+    position: absolute;
+    height: 100%;
+    pointer-events: none;
+  }
```

Shouldn't have the same styles as x-input?

```suggestion
  > .x-input-placeholder {
    flex: 1 1 100%;
    min-width: 0;
    border: none;
    position: absolute;
    pointer-events: none;
  }
```